src/neuroconv/datainterfaces/ecephys/intan/intandatainterface.py

```diff
@@ -78,8 +78,14 @@ def __init__(
             check performed is that timestamps are continuous. If False, an error will be raised if the check fails.
         """
 
-        neo_version = get_package_version(name="neo")
-        spikeinterface_version = get_package_version(name="spikeinterface")
+        if stream_id is not None:
+            warnings.warn(
+                "Use of the 'stream_id' parameter is deprecated and it will be removed after September 2024.",
+                DeprecationWarning,
+            )
+            self.stream_id = stream_id
```

Comment:
Is the plan to replace this with `stream_name`?

Reply:
Ah, just moving this up from below...

still a valid question though lol, can you remind me?

Reply:
No, the plan is to fix it because stream_id=0 are the amplifiers channels.